### PR TITLE
Moonrank collections indexing

### PR DESCRIPTION
**Pull request by imabdulbasit**

### CODE REVIEW

File: crates/moonrank/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+[[bin]]
+name = "moonrank-collections-indexer"
+path = "main.rs"
+
+[dependencies]
+reqwest = { version = "0.11.6", features = ["json", "gzip", "brotli", "deflate"] }
+url = "2.2.2"
+serde = { version = "1.0.130", features = ["derive"] }
+serde_json = "1.0.70"
+tokio = { version = "1.13.0", features = ["macros", "rt", "rt-multi-thread", "signal", "sync", "full"] }
```
[the remark this answers is not in the document]
Comment (imabdulbasit) [reply crates/moonrank/Cargo.toml]: I was trying something else which required more features. I am not using most of these

File: crates/core/src/db/models.rs [hunk 2]
```diff
@@ -3376,3 +3376,41 @@ pub struct ProposalTransactionInstructionAccount<'a> {
...
+pub struct Collection<'a> {
+    pub id: Cow<'a, str>,
+    pub image: Cow<'a, str>,
+    pub name: Cow<'a, str>,
+    pub description: Cow<'a, str>,
+    pub verified_collection_address: Option<Cow<'a, str>>,
+    pub pieces: i64,
+    pub verified: bool,
+    pub created_at: NaiveDateTime,
+    pub updated_at: NaiveDateTime,
```
Comment (kespinola) [inline crates/core/src/db/models.rs:3392]: There are some additional fields that I'd like to index. Can you include this set please?

```
id [primary key]
image 
name
description
twitter_url
discord_url
website_url
magic_eden_identifier [index]
verified
pieces
go_live_at
verified_collection_address
created_at
updated_at
```

**Review by kespinola — CHANGES_REQUESTED** [review]
Looks great just the additional fields to save on collections. Thank you.

**Review by kespinola — COMMENTED** [review]
Looks good. Seems like all we need now is to build the binary in the docker file and add its docker script for the startup.

**Review by mpwsh — DISMISSED** [review]

**Review by kespinola — APPROVED** [review]

**Review by mpwsh — APPROVED** [review]

**Pull request merged**